cpp/daal/include/services/internal/sycl/math/blas_executor.h

```diff
@@ -200,10 +208,14 @@ class SyrkExecutor
 public:
     static void run(cl::sycl::queue & queue, const math::UpLo upper_lower, const math::Transpose trans, const size_t n, const size_t k,
                     const double alpha, const UniversalBuffer & a_buffer, const size_t lda, const size_t offsetA, const double beta,
-                    UniversalBuffer & c_buffer, const size_t ldc, const size_t offsetC, services::Status & status)
+                    UniversalBuffer & c_buffer, const size_t ldc, const size_t offsetC, Status & status)
     {
-        Execute op(queue, upper_lower, trans, n, k, alpha, a_buffer, lda, offsetA, beta, c_buffer, ldc, offsetC, status);
-        TypeDispatcher::floatDispatch(a_buffer.type(), op);
+        DAAL_ASSERT(!a_buffer.empty());
+        DAAL_ASSERT(!c_buffer.empty());
+        DAAL_ASSERT(a_buffer.type() == c_buffer.type());
```

Comment:
buffer size check?

Reply:
This check is going to be performed in `Executor`